src/openrct2/rct1/T4Importer.cpp

```diff
@@ -25,271 +25,275 @@
 /**
  * Class to import RollerCoaster Tycoon 1 track designs (*.TD4).
  */
-class TD4Importer final : public ITrackImporter
+namespace OpenRCT2
```

Comment:
Shouldn't this be in the namespace of `OpenRCT2::RCT1`, then?

Reply:
I could see it being in either namespace. With the file currently being in the rct1 directory, I'll try changing the namespace and see how it changes the rest of the code it imports.